src/UNL/UCBCN/Calendars.php

```diff
@@ -0,0 +1,39 @@
+<?php
+namespace UNL\UCBCN;
+
+use UNL\UCBCN\ActiveRecord\RecordList;
+use UNL\UCBCN\ActiveRecord\Record;
+/**
+ * Object related to a list of calendars.
+ * 
+ * PHP version 5
+ * 
+ * @category  Events 
+ * @package   UNL_UCBCN
+ * @author    Tyler Lemburg <trlemburg@gmail.com>
+ * @copyright 2015 Regents of the University of Nebraska
+ * @license   http://www1.unl.edu/wdn/wiki/Software_License BSD License
+ * @link      http://code.google.com/p/unl-event-publisher/
+ */
+
+class Calendars extends RecordList
+{
+    public function getDefaultOptions() {
+        return array(
+            'listClass' =>  __CLASS__,
+            'itemClass' => __NAMESPACE__ . '\\Calendar',
+        );
+    }
+
+    public function getSQL() {
+        if (array_key_exists('account_id', $this->options)) {
```

Comment:
sounds good to me.  :)  If you aren't checking for NULL, you might want to handle the possibility of it being NULL.  I think this code might just convert it to an empty string, which I don't think would be a big deal.